### PR TITLE
design home page

### PR DESCRIPTION
**Pull request by intern-tieumomo**
## Related Tickets:
- [#28869](https://edu-redmine.sun-asterisk.vn/issues/28869)

## WHAT this PR does?
- Design home page

## Notes Impacted Areas:
- N/A.

## Attachments:
- screencapture-localhost-8000-2020-09-10-12_07_43

### CODE REVIEW

**Comment by intern-tieumomo** [comment]
ready

File: .bowerrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"directory": "public/bower_components/"
+}
```
Comment (huyenttm-0901) [inline .bowerrc]: end of file

File: app/Http/Controllers/HomeController.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class HomeController extends Controller
+{
+    public function index()
+    {
+    	return view('home');
+    }
```
Comment (huyenttm-0901) [inline app/Http/Controllers/HomeController.php]: indent

File: resources/views/layouts/app.blade.php [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                    <div class="col-12 col-md-9 order-md-2 order-lg-3 col-lg-3">
+                        <div class="header-right-wrap">
+                            <ul>
+                                <li><a href="">LOGIN</a></li>
+                                <li><a href="">REGISTER</a></li>
+                                <li class="header-search"><a class="header-search-toggle" href="#"><i class="icofont-search-2"></i></a>
+                                    <div class="header-search-form">
+                                        <form action="#">
+                                            <input type="text" placeholder="Type and hit enter">
+                                            <button><i class="icofont-search"></i></button>
```
Comment (huyenttm-0901) [inline resources/views/layouts/app.blade.php]: i18n

File: resources/views/layouts/app.blade.php [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+                    </div>
+                </div>
+                
+            </div>
+        </div>
+    </header>
+    
+    @yield('content')
+
+    <footer class="footer-section style-2 section bg-theme" style="background-image: url({{ asset('images/bg/footer-bg.jpg') }}">
```
Comment (huyenttm-0901) [inline resources/views/layouts/app.blade.php]: viết riêng css

File: resources/views/layouts/app.blade.php [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+                </div>
+            </div>
+        </div>
+        
+        <div class="footer-bottom border-color section">
+            <div class="container">
+                <div class="row">
+                    <div class="col-12">
+                        <div class="copyright text-center">
+                            <a target="_blank" href="https://www.templateshub.net">Templates Hub</a>All rights reserved.</p>
```
Comment (huyenttm-0901) [inline resources/views/layouts/app.blade.php]: ko dùng link online

**Comment by intern-tieumomo** [comment]
ready

**Pull request merged**